helm-chart/renku/templates/network-policies.yaml

```diff
@@ -991,6 +991,33 @@ spec:
 ---
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
+metadata:
+  # Needed for secret mounting
+  name: ingress-to-data-service-from-v2-sessions
```

Comment:
This means the data service can be access on a cluster IP from within any user session?

Reply:
Yes, it is already the case for v1 sessions.